_data/infrastructure-projects.yaml

```diff
@@ -0,0 +1,31 @@
+- projects:
+  - name: Limpopo region
+    coordinates: []
+    stage: Construction
+    total_budget: 142200000000
+    department: Water and Sanitation
+    description: Construction of water supply and sanitation backlog
+    province: ??? how do we determine the province? it is not always in the project name
+    projected_budget: 51800000000
+    detail: /path/to/detail/page
+  - name: Police stations
+    coordinates:
+      - -30.4759, 30.2030
+    stage: Design
+    total_budget: 142200000000
+    department: Police
+    description: Construction of new and re-established police stations
+    province: ??? how do we determine the province? it is not always in the project name
```

Comment:
get province from coordinate

Reply:
https://mapit.code4sa.org/#api-by_point

Reply:
something like https://mapit.code4sa.org/point/-24.898520/29.757140 whatever the API endpoint is

Reply:
http://mapit.code4sa.org/point/4326/29.757140,-24.898520.html

Reply:
this probably needs to be an array since there could be more than one :grin: